New libs: Change permission requirement to notification

srittau opened the pull request:
This was discussed on the typing-sig mailing list and in
python/peps#1218, has met the approval of the steering council, and
was incorporated into PEP 484.

PEP 484 does not require the submitter to actively contact the library
owners, but I believe we should require this courtesy.

JelleZijlstra commented on CONTRIBUTING.md:
```diff
@@ -107,14 +107,13 @@ with what you'd like to do or have ideas that will help you do it.
 If you want to submit type stubs for a new library, you need to
-**contact the maintainers of the original library** first to let them
-know and **get their permission**.  Do it by opening an issue on their
+**contact the maintainers of the original library** to let them
```
I have been thinking whether it would be better to have us (the typeshed maintainers) do the notification instead, so we can make it even easier for contributors to contribute stubs without worrying about formalities. What do you think?

srittau replied:
Good idea. We could use a standard template for that, which could contain more information than another contributor would normally provide. (Links to projects, the "type school" if that happens etc.)

brettcannon replied:
If people feel it's important to do the notification then having typeshed do it would be the lowest barrier to entry, but it won't scale if this catches on, so what I would do is remove the requirement and just make it a best practice that is okay to drop if it turns out to be too much of a burden.

srittau commented:
I have updated the PR and just removed the "Adding a new library section". For now, I have also added a recommendation for core developers to contact the original maintainers. If it turns out to be too big a burden, we can remove that recommendation per @brettcannon's suggestion.

We should add a notification template in the future, but I want to keep this PR focused.
